src/ParseObject.js

```diff
@@ -935,10 +939,22 @@ class ParseObject {
   }
 
   /**
-   * Clears any changes to this object made since the last call to save()
+   * Clears any (or specific) changes to this object made since the last call to save()
+   * @param {String|Array<String>|...String} [keys] - specify which fields to revert
```

Comment:
This is very confusing, it’s either an array or a list of arguments, but all at once a makes it hard to reason about and potentially buggy